templates/misc/embedded-content-cookie-compliance.twig

```diff
@@ -1,5 +1,5 @@
 <div class="embedded-content-cookie-compliance media-{{ media_id }}">
-  <div class="message">
+  <div class="message" {{ alternative_language ? create_attribute({'lang': 'en', 'dir': 'ltr'}) }}>
```

Comment:
Should we maybe move the `en` and `ltr` to a common place, so that it's not defined in million places, just in case someone wants to have another default language?

Reply:
I actually now switched them to be `$variables['lang_attributes']['fallback_lang/fallback_dir]` because if they are inside `['attributes']` they would be added to everything where attributes are added in the twig files. 😬